addons/pronelauncher/functions/fnc_onKeyDown.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+ * Author: PiZZADOX, Jonpas
+ * Handles keyDown EH for overriding engine stance changes when in AT launcher stance.
+ *
+ * Arguments:
+ * 0: Control <CONTROL>
+ * 1: Key <NUMBER>
+ * 2: Shift <BOOL>
+ * 3: Ctrl <BOOL>
+ * 4: Alt <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [control, 5, false, true, false] call ace_pronelauncher_fnc_onKeyDown
+ *
+ * Public: No
+ */
+
+params ["","_key"];
+
+if !(isNull objectParent ACE_player) exitWith {false};
+if (currentWeapon ACE_player != secondaryWeapon ACE_player) exitWith {false};
+
+private _stance = stance ACE_player;
+private _keysMoveDown = actionKeys "moveDown";
+private _keysMoveUp = actionKeys "moveUp";
```

Comment:
Use of `actionKeys` command is unacceptable, because it reports out of precission range for single point floats for keybinds with modifier or hold or 2x binds.

Reply:
what would be the solution to this then? Have players who want to use the feature unbind their stance change keys and have a cba keybind?